From abdf03db214224f907515c3309634a3b13cdc605 Mon Sep 17 00:00:00 2001
From: Philippe Coval <philippe.coval@vates.tech>
Date: Mon, 15 Dec 2025 11:26:31 +0100
Subject: [PATCH] s/c/e/profile.d/history.sh: Add timestamps to history

... using HISTTIMEFORMAT

The ISO prefix is used because easier to read/parse and less prone to
ambuiguity (EU vs US).

The file structure is aligned to xs.sh

Signed-off-by: Philippe Coval <philippe.coval@vates.tech>
---
 src/common/etc/profile.d/history.sh | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 src/common/etc/profile.d/history.sh

diff --git a/src/common/etc/profile.d/history.sh b/src/common/etc/profile.d/history.sh
new file mode 100644
index 0000000..999c80a
--- /dev/null
+++ b/src/common/etc/profile.d/history.sh
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+HISTTIMEFORMAT="%Y-%m-%d %T "
+export HISTTIMEFORMAT
